.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,6 +7,13 @@ assignees: ''
 
 ---
 
+**ATTENTION**
+
+Please start filing all issues in our new repo: [https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/new/choose.](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/new/choose)
```

Comment:
This could be a little surprising for people to first learn about the new repo from here.
We should probably also have a blurb about the new repo on the front-page readme.
Can we just say, if this is an issue for version 2.0.0 or higher, please file it [here](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/new/choose)?
```suggestion
If this is an issue for version `2.0.0` or higher, please file it in our new repo: [GoogleCloudPlatform/spring-cloud-gcp](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/new/choose).
```

Reply:
Well remember:
 a.) not many external folks are filing issues these days (so relatively low impact), 
b.) we're only a few weeks away from releasing 2.0, 
c.) the Spring Cloud announcement that this is happening is almost 18 months old, 
d.) all devs should be cherry-picking fixes to both repos anyway (doesn't matter if the bug is 2.0 or not), and 
e.) Issues in the new repo fall under our new SLO.

Reply:
Some good points, but in that case, "Please start filing..." sounds a bit weak.
Wouldn't we want to say then, "Please file all issues in the new repo."?

Reply:
Done.